src/SIL.XForge.Scripture/Services/ParatextService.cs

```diff
@@ -228,11 +228,19 @@ public void Init()
             if (ptProject is not ParatextResource)
             {
                 StartProgressReporting(progress);
+
+                string username = GetParatextUsername(userSecret);
+                // Specifically set the ScrText property of the SharedProject to indicate the project is available locally
```

Comment:
This comment feels like it is now out of place. I can't make sense of what it's trying to communicat here, though it looks like it may have made some sense in its previous context.

Reply:
Good observation - I've moved it to a more sensible place.